mlflow/ml-package-versions.yml

```diff
@@ -211,13 +211,16 @@ fastai-1.x:
     pip_release: "fastai"
 
   models:
+    # Avoid updating the maximum version to 2.x which isn't supported yet
+    freeze: True
```

Comment:
@dbczumar Thanks for [this comment](https://github.com/mlflow/mlflow/pull/4174#pullrequestreview-654748537), I  added this field `freeze`. When this field is present, the script doesn't skips updating the maximum version. WDYT?

Reply:
Nit: call the field "frozen" ?